irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -1286,13 +1286,30 @@ namespace iroha {
           {"createAccount",
            createAccountBase,
            {(boost::format(R"(
-            has_perm AS (%s),)")
+           domain_role_permissions_bits AS (
+                 SELECT COALESCE(bit_or(rhp.permission), '0'::bit(%1%)) AS bits
+                 FROM role_has_permissions AS rhp
```

Comment:
Please, give a little more meaningful names to `rhp` and `rp`

Reply:
Discussed. Will use rhp in all places